### DISCUSSION TITLE
Fix the notorious Exited with code: 1 message

### DISCUSSION THREAD

**Discussion by satyavh** [meteor/meteor#11769] [Feature Requests]
It would be great if this notorious Meteor error message is providing information

```
Exited with code: 1 
=> Your application is crashing. Waiting for file change.
```

It almost always means an import can't be resolved during building, but it should give a decent message which import it can't resolve. I lost count of the hours trying to figure out where the application crashes.

So please add better error message!

**Comment by vitorflores** [comment]
I don't think we show this message without any explanation or error detail.

Did you reproduce it?

It seems more an issue (and maybe an invalid one) than a feature request.

I think you can convert to an issue and ask a reproduction. In case you can't reproduce with a simple test using an invalid import.

**Reply by satyavh** [reply]
This might actually be happening when using Meteor + Vue package, and is thrown by that package. I'll dive into it
